app/templates/formbuilder/edit/code.hbs

```diff
@@ -17,6 +17,12 @@
       >
         {{t "codeConsole.buttonNotification"}}:
         {{this.consoleMessages.length}}
+        {{#if this.ttlHasErrors}}
+          <AuPill @skin="error" @icon="cross">Error</AuPill>
+        {{/if}}
+        {{#if this.ttlHasWarnings}}
+          <AuPill @skin="warning" @icon="cross">Warning</AuPill>
+        {{/if}}
       </AuButton>
```

Comment:
Adding pills in a button?

Could you move the pills to the right side of the console bar? 
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/bc64236a-27e0-44a3-8d31-d9928cc2a215)

I would do or text or the icon in the pill but not both